src/popup/router/components/TransactionList.vue

```diff
@@ -21,7 +21,7 @@
       data-cy="loader"
     />
     <div
-      v-else-if="!transactions.length"
+      v-else-if="!filteredTransactions.length"
```

Comment:
How can I reproduce it? I have this message on newly created account on develop version:
![image](https://user-images.githubusercontent.com/7098449/126448663-4dff1e0c-6129-42c2-aebe-7dd7baf69dd2.png)

Reply:
Ok, got it, it's after filtering